### PR TITLE
Fix one-way video notification

### PR DESCRIPTION
**Pull request by gugalo**
One way-video call notification was saying "Operator can see you" because there was no separate message for one-way calls

- Added one-way video call notification message
- Updated all audio and video notifications with new messages provided by Alina and approved by Henrik
- Combined all audio and video notifications Use Cases to one to make code a bit more maintainable

**Jira issue:**
https://glia.atlassian.net/browse/MOB-2090

**Call Visualizer Screenshots:**

image

image

**Regular Engagement Screenshots:**
image

image

image

**Screen-sharing notification Screenshot:**

image

### CODE REVIEW

**Review by DavDo — APPROVED** [review]

**Review by Pelkar — APPROVED** [review]
LGTM 👍

File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+) {
+    operator fun invoke(visitorMedia: VisitorMediaState? = null, operatorMedia: OperatorMediaState? = null) {
+        val audio: MediaDirection
+        val video: MediaDirection
+        try {
+            audio = getAudioDirection(visitorMedia, operatorMedia)
+            video = getVideoDirection(visitorMedia, operatorMedia)
+        } catch (error: IllegalStateException) {
+            // One of the impossible scenario was detected during parsing of the states which is not supported by current GliaHub
+            // TODO: should it throw exception or return error or just silently log warning
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt:19]: Regarding this TODO. I would vote for a silent error log. We should notice it in Sentry and take action.

**Review by andrews-moc — APPROVED** [review]
Combined audio and video notifications Use Cases into a single one look good to me. 👍

File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt (hunk 1 above)
Comment (gugalo) [reply widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt:19]: Oh, right!
Forgot about this one. Thank you!
I will add a silent error log. But unless something changed lately in Widgets SDK we don't yet send the errors to the server. Only the Core SDK does that

Comment (gugalo) [reply widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/CallNotificationUseCase.kt]: Fixed in 5716d01

**Comment by gugalo** [comment]
!merge

**Pull request merged**